<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register_Widget</class>
 <widget class="QWidget" name="Register_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;
border-image: url(:/backcaption.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="path">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>存储路径</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dircomboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="search">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="code_input">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="recode_input">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="code">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="recode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QToolButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>104</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/confirm_button.png);
font: 24pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>进  入</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>105</width>
     <height>105</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>311</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>452</width>
     <height>254</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>452</width>
     <height>254</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/decode.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="decode_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>505</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DECODE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="confirm_close">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>104</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/confirm_button.png);
font: 24pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>确  认</string>
   </property>
  </widget>
  <zorder>confirm_close</zorder>
  <zorder>confirm</zorder>
  <zorder>back</zorder>
  <zorder>path</zorder>
  <zorder>dircomboBox</zorder>
  <zorder>search</zorder>
  <zorder>name_input</zorder>
  <zorder>code_input</zorder>
  <zorder>recode_input</zorder>
  <zorder>name</zorder>
  <zorder>code</zorder>
  <zorder>recode</zorder>
  <zorder>picture</zorder>
  <zorder>info</zorder>
  <zorder>warning</zorder>
  <zorder>decode_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
